app/assets/stylesheets/main.css.scss

```diff
@@ -19,4 +19,8 @@ hr{
 
 .home-index-pins{
   margin-top: 56px; 
+}
+
+.scrollControl{
```

Comment:
CSS classes should use [spinal-case](http://en.wikipedia.org/wiki/Letter_case), not camelCase